source/Pages/compute_on_spider.rst

```diff
@@ -305,7 +318,408 @@ combination that is not available you will receive the following error message:
 
    'sbatch: error: Batch job submission failed: Requested node configuration is not available'
 
+===============
+Using GPU nodes
+===============
+
+To run your program on GPU nodes some guidelines for the user have to be taken into account. Firstly, GPUs are only available on the GPU nodes ``wn-gp-[01,02]`` and ``wn-ga-[01,02]`` and **not** on the UI nodes. All GPU nodes run Nvidia hardware and as such, CUDA software is necessary. The CUDA drivers are installed on the relevant machines, but the CUDA interface and other programs need to be run in a singularity container. Nvidia has containers available on the internet for CUDA use, that can be built upon. These can be found `here <https://catalog.ngc.nvidia.com/containers>`_. Alternatively, you can build your own container from scratch, which is also shown in the next section. Before building the container, the user needs to know the version of the drivers available on the GPU nodes. The version can be found with:
```

Comment:
1) You state that "GPUs are only available on the GPU nodes" this may seem somewhat obvious. Perhaps we should state that "GPU (cuda) drivers are only available on the GPU nodes"
2) "and as such,  CUDA software is necessary" this is not true and I would suggest to remove that part of the sentence
3) "The CUDA drivers are installed on the relevant machines, but the CUDA interface and other programs need to be run in a singularity container." this is not true. I suggest "CUDA drivers are available on the GPU nodes. Other GPU software needs to be obtainer and deployed by users. We suggest users to create [link] or make use of pre-build Singularity containers (e.g., [link to NVIDIA repo and perhaps other repos]). The Spider team can provide assistance to users who are not familiar with container. In this case please submit your request for assistance via our [helpdesk link]"
4) Why does a user need to know the version of the available drivers?